libpod/options.go

```diff
@@ -283,14 +283,15 @@ func WithCNIPluginDir(dir string) RuntimeOption {
 
 // Container Creation Options
 
-// WithShmDir sets the directory that should be mounted on /dev/shm
-func WithShmDir(dir string) CtrCreateOption {
+// WithShmDir sets the directory that should be mounted on /dev/shm and the size of it
+func WithShmDir(dir string, size int64) CtrCreateOption {
```

Comment:
Can you separate size into a separate option? I can see people wanting the default location we use but a custom size or vice versa

Reply:
Done